ISSUE_TEMPLATE

```diff
@@ -0,0 +1,40 @@
+Before filing an issue:
+-----------------------
+- Attempt to reproduce the issue using 
+  [this sample application](https://github.com/forcedotcom/ServiceSDK-Android/)
+- Search existing issues, including issues that are closed.
```

Comment:
What about making "issues" a link to the right spot:
`Search [existing issues](https://github.com/forcedotcom/ServiceSDK-Android/issues), including issues that are closed.`

Reply:
👍